sys/include/libsec.h

```diff
@@ -7,8 +7,6 @@
  * in the LICENSE file.
  */
 
-
-
 #ifndef _MPINT
```

Comment:
No. We don't do this in harvey, we follow the Plan 9 standards. This #ifdef has to go.